elated-egrets/bangazon-internal-site
# J rfield

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRfield
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. run the server and make sure that the date field and training program field are not required.

## What you changed
made the fields for end date and training programs not required
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
